README.md

```diff
@@ -27,6 +28,11 @@ After being first registered, you need to execute the folloing command to promot
 
 and respond yes to the question ` Add admin privileges to this user?`
 
+### Pixelfed php commands and php version
+
+Pixelfed might require some command line instructions if you make manual changes to your configuration.
```

Comment:
It seems that in a near future manual changes of the config file won't be necessary as most of parameters will be available from the Admin panel. 

So or we don't add this paragraph or it will be interesting to add the link to the configuration instruction here and explain what command to use ;)

Reply:
Well, I guess there will still be some cases were people would like (need ?) to do that. For instance command line specific features (I do know, maybe cache clearing, this kind of stuff).
So I think it's still useful to keep it.